src/components/molecules/Wallet/Details.tsx

```diff
@@ -62,9 +96,11 @@ export default function Details(): ReactElement {
         ))}
 
         <li className={styles.actions}>
-          <span title="Connected provider">
-            <img className={styles.walletLogo} src={providerInfo?.logo} />
-            {providerInfo?.name}
+          <span title="Connected provider" className={styles.walletInfo}>
+            <div>
```

Comment:
a block-level HTML tag within an inline-level tag is invalid HTML. Ideally, remove the `div` and see if it works with less markup, otherwise switch the `span` & `div` tags